AvatarFallback.mjs issue on production.

### Describe the bug

Hello!

On production I have an issue with AvatarFallback.mjs

Cannot find module '/app/.output/server/node_modules/@oku-ui/primitives/dist/node_modules/.pnpm/@vueuse_shared@11.2.0_vue@3.5.13_typescript@5.6.3_/node_modules/@vueuse/shared/index.mjs' imported from /app/.output/server/node_modules/@oku-ui/primitives/dist/avatar/AvatarFallback.mjs

Nuxt 3.14.1592 with SSR, on first load.
I also have installed @vueuse/motion.

In Reproduction do the build and run npx nuxi preview (node .output/server/index.mjs)

### Reproduction

https://stackblitz.com/edit/nuxt-starter-yw2xh5?file=app.vue

### System Info

```Shell
System:
    OS: macOS 15.1
    CPU: (12) arm64 Apple M3 Pro
    Memory: 230.42 MB / 18.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 21.5.0 - ~/.nvm/versions/node/v21.5.0/bin/node
    npm: 10.2.4 - ~/.nvm/versions/node/v21.5.0/bin/npm
    pnpm: 9.1.4 - ~/.nvm/versions/node/v21.5.0/bin/pnpm
    bun: 1.0.20 - ~/.bun/bin/bun
  Browsers:
    Brave Browser: 125.1.66.118
    Chrome: 130.0.6723.117
    Edge: 121.0.2277.83
    Safari: 18.1
```


### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/productdevbook/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/packages/plugins/src/nuxt/index.ts
+++ b/packages/plugins/src/nuxt/index.ts
@@ -50,7 +50,7 @@ export default defineNuxtModule<ModuleOptions>({
       addComponent({
         name: `${options.prefix}${component}`,
         export: component,
-        filePath: '@oku-ui/primitives',
+        filePath: '@oku-ui/motion',
       })
     }
 

--- a/packages/plugins/src/resolver/index.ts
+++ b/packages/plugins/src/resolver/index.ts
@@ -21,7 +21,7 @@ export default function (options: ResolverOptions = {}): ComponentResolver {
         if (Object.values(components).flat().includes(componentName)) {
           return {
             name: componentName,
-            from: '@oku-ui/primitives',
+            from: '@oku-ui/motion',
           }
         }
       }

